src/shared/style/form.css

```diff
@@ -52,4 +52,15 @@
 .form-btn:hover {
     background-color: var(--color-white);
     color: var(--color-black);
+}
+
+.form-btn:disabled {
+    background-color: var(--color-primary);
+    color: var(--color-white);
+    opacity: .7;
+    cursor: not-allowed;
+}
+
+.form-error {
+    color: red;
```

Comment:
Since you'll use `ValidationMessage` component, we already have the style for the validation message.